### ISSUE TITLE
Add minimap to cammie

### ISSUE THREAD

**Issue by werthen** [ZeusWPI/zeus.ugent.be#23]
We can give absolute coordinates to cammie. Why don't we add a little minimap to the cammie interface so you can immediately focus on a certain point?

**Comment by werthen** [comment]
@redfast00 why close this?

**Comment by redfast00** [comment]
I thought the buttons (zetel, grote tafel, kleine tafel) did this.

**Comment by werthen** [comment]
Yes, they provide easy access to the primary locations, but what i meant was have an actual minimap (so a picture) where you can click anywhere and it would view that location. I agree with closing this however because it's a bit unnecessary

**Comment by wschella** [comment]
I don't agree, the issue-list is full of unnecessary shit, but it's _cool_.
